internal/util/projutil/project_util.go

```diff
@@ -55,6 +55,8 @@ const (
 	OperatorTypeAnsible OperatorType = "ansible"
 	// OperatorTypeHelm - helm type of operator.
 	OperatorTypeHelm OperatorType = "helm"
+	// OperatorTypeHybrid - hybrid type of operator.
+	operatorTypeHybrid OperatorType = "hybrid"
```

Comment:
can we make this `hybrid.helm`/`hybrid+helm`/`hybrid-helm` or something like that, since it's possible we'll see more hybrid plugins in the future?